charts/neon-storage-scrubber/templates/cronjob.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "neon-storage-scrubber.fullname" . }}
+  labels:
+    {{- include "neon-storage-scrubber.labels" . | nindent 4 }}
+spec:
+  timeZone: "Etc/UTC"
+  schedule: '0 18 * * *' # Runs every day at 6pm
+  failedJobsHistoryLimit: 1
+  concurrencyPolicy: Forbid # Only runs one instance
+  jobTemplate:
+    metadata:
+      name: {{ include "neon-storage-scrubber.fullname" . }}
+      labels:
+        {{- include "neon-storage-scrubber.labels" . | nindent 8 }}
+    spec:
+      backoffLimit: 0 # Do not restart if the task fails
+      template:
+        metadata:
+          {{- with .Values.podAnnotations }}
+          annotations:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          labels:
+            {{- include "neon-storage-scrubber.selectorLabels" . | nindent 12 }}
+        spec:
+          restartPolicy: Never # Do not restart if the task fails
+          {{- with .Values.imagePullSecrets }}
+          imagePullSecrets:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          serviceAccountName: {{ include "neon-storage-scrubber.serviceAccountName" . }}
+          securityContext:
+            {{- toYaml .Values.podSecurityContext | nindent 12 }}
+          containers:
+            - name: {{ .Chart.Name }}
+              securityContext:
+                {{- toYaml .Values.securityContext | nindent 16 }}
+              image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+              imagePullPolicy: {{ .Values.image.pullPolicy }}
+              command:
+                - /usr/local/bin/storage_scrubber
+                - pageserver-physical-gc
+                - --min-age=1week
```

Comment:
can you make all cli args configurable?

Reply:
are we really need it configurable ? or you about min age option ?

Reply:
in case if need it then we don't have to refactor helm chart and repeat all process, so it will be helpful if its configurable from value file. I faced this issue in many helm chart that doesn't support this, so you end-up adding patches to fix it.

Reply:
> are we really need it configurable ? or you about min age option ?

I don't mean only `--min-age` but make it configurable any cli option, so that we can modify from values not from template that requires releasing new helm chart for any cli flag modification.